ansible/roles/gen_kubernetes_config/templates/forkmon.yaml.j2

```diff
@@ -0,0 +1,64 @@
+# ANSIBLE GENERATED FILE - DO NOT EDIT MANUALLY
+forkmon:
+  fullnameOverride: forkmon
+  image:
+    repository: skylenet/nodemonitor
```

Comment:
lol, think this has been merged even

Reply:
also maybe move all the images to ansible `images.yaml` group var, easier to update it there then. Esp since its generating this config either ways.